style.css

```diff
@@ -1,16 +1,48 @@
-#number-guess,
-#result,
-#history {
-  max-width: 400px;
-  margin: 0 auto;
+#number-guess, #result, #history {
+    max-width: 400px;
+    margin: 0 auto;
 }
 
 #history {
-  margin-top: 16px;
+    margin-top: 16px;
+    max-height: fit-content;
 }
 
 .banner {
-  color: #ffd23f;
-  font-size: 90px;
-  font-family: sans-serif;
+    color: yellow;
+    font-size: 45px;
+    font-family: sans-serif;
+    text-decoration: solid;
+    font-weight: 600;
+}
+
+main {
+    padding: 100px;
+}
+
+body {
+    background-image: url("Images/BG1.png");
+    background-size: fill;
 }
+
+.gameBG {
+    background-color: black;
+    border: 1px solid black;
```

Comment:
- [ ] On the background image, you have given 1px solid black border. Try saying no border and see if it looks odd. Otherwise, it is fine just leave it.

Reply:
done

Reply:
So was it good or bad?